test/downloadSpecTests.ts

```diff
@@ -8,9 +8,10 @@ downloadTests(
     specVersion: SPEC_TEST_VERSION,
     outputDir: SPEC_TESTS_DIR,
     testsToDownload: SPEC_TEST_TO_DOWNLOAD,
+    specTestsRepoUrl: "https://github.com/ethereum/consensus-spec-tests",
   },
   console.log
-).catch((e) => {
+).catch((e: any) => {
```

Comment:
why is this needed, we should also avoid using any..this repo needs proper linting rules